package.json

```diff
@@ -1,11 +1,13 @@
 {
-  "homepage": "./",
+  "homepage": "/",
```

Comment:
Why this change?

Reply:
I am not really sure, I think this was a mistake so have changed it.